cfn/README-GD.md

```diff
@@ -132,40 +97,33 @@ regions and accounts](https://docs.aws.amazon.com/guardduty/latest/ug/guardduty_
 
 ### Use the Amazon console to deploy
 
-**Note:** This procedure assumes setup in the AWS `us-east-1` region using the Alert Logic Cloud Insight [US console](https://console.cloudinsight.alertlogic.com/#/login). If your setup is in a European region (e.g., `eu-east-1`), use the [UK console](https://console.cloudinsight.alertlogic.co.uk/#/login).
+**Note:** This procedure assumes setup in the AWS us-east-1 region through the Cloud Insight Essentials [US console](https://console.cloudinsight.alertlogic.com/#/login). If your setup is in an AWS European region, like eu-east-1, use the [UK console](https://console.cloudinsight.alertlogic.co.uk/#/login).
 
 1. Log in to the [AWS Management Console](https://aws.amazon.com/console/) with an AWS account that has AWS administrator privileges. 
 1. Select the region in which you want to deploy the CFT.
-1. Click **Services**->**CloudFormation**->**Create Stack**.
-1. In the `Choose a template` section select **Specify an Amazon S3 template URL** and enter the following URL.
+1. Click **Services** > **CloudFormation** > **Create Stack**.
+1. In the **Choose a template** section, select **Specify an Amazon S3 template URL**, and then enter the following URL"
 `https://s3.amazonaws.com/alertlogic-collectors-us-east-1/cfn/guardduty.template`
 1. Click **Next**.
-1. On the `Specify Details` window, provide the following required parameters:
-   - `Stack name` - Any name you have used for creating an AWS stack
+1. In the **Specify Details** window, provide the following required parameters:
+   - `Stack name` - Any name you used to create an AWS stack
    - `AccessKeyId` - `access_key_id` returned from AIMs [above](#alert-logic-access-key-creation)
    - `AlApiEndpoint` - usually `api.global-services.global.alertlogic.com` 
    - `AlDataResidency` - usually `default`
    - `SecretKey` - `secret_key` returned from AIMs [above](#alert-logic-access-key-creation)
 1. Click **Next**. 
-1. On the `Options` panel, click **Next**.
-1. In the `Review` panel, perform a predeployment check. 
+1. On the **Options** panel, click **Next**.
+1. In the **Review** panel, perform a predeployment check. 
 1. Select **"I acknowledge that AWS CloudFormation might create IAM resources"**, then click **Create**.
-1. On the CloudFormation, `Stacks` panel, filter based on the stack name you created, and then 
-select your stack by name.
+1. On the CloudFormation, **Stacks** panel, filter results based on the stack name you created, and then 
+select your stack.
 
 If the deployment was successful, the status appears as: CREATE_COMPLETE. If the deployment was not successful, 
 see [Troubleshooting Installation Issues](#troubleshooting-installation-issues) below.
 
-**Note:** Only one collector installation is allowed per AWS region. If you try to deploy the template multiple times in the same region, if will fail with the following error:
```

Comment:
I think is still important to highlight that only one CFT deployment is allowed per region.
Or move it to `Troubleshooting` section.

Reply:
I'll restore the note, and change the language a bit.